webapp/src/components/channel_call_toast.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useMemo, useState} from 'react';
+import {useIntl} from 'react-intl';
+import {useSelector} from 'react-redux';
+
+import {Client4} from 'mattermost-redux/client';
+import {getCurrentChannelId} from 'mattermost-redux/selectors/entities/channels';
+
+import ConnectedProfiles from 'src/components/connected_profiles';
+import ActiveCallIcon from 'src/components/icons/active_call_icon';
+import {useDismissJoin} from 'src/components/incoming_calls/hooks';
+import Timestamp from 'src/components/timestamp';
+import {
+    connectedChannelID, isLimitRestricted,
+    voiceChannelCallInCurrentChannel, voiceConnectedCurrentChannel, voiceProfilesInCurrentChannel,
+} from 'src/selectors';
+import {callStartedTimestampFn} from 'src/utils';
+
+const ChannelCallToast = () => {
```

Comment:
Not sure if it's a regression but I noticed that if you click to close the toast and a new call starts on the same channel you don't get it back, although my expectation is that you should.

Reply:
Looks like it's that way on main too -- you have to change channels for it to come back. I made a ticket for it, and I'll take care of it: https://mattermost.atlassian.net/browse/MM-53322